devicetypes/smartthings/eaton-dimmer.src/eaton-dimmer.groovy

```diff
@@ -0,0 +1,333 @@
+/**
+ *  Eaton Accessory Dimmer
+ *
+ *  Copyright 2017 Smartthings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *  	http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition (name: "Eaton Dimmer", namespace: "smartthings", author: "Smartthings") {
```

Comment:
Have you tried setting this to run locally? I think it should work with `runLocally: true, executeCommandsLocally: false`